Region refresh on a disconnected cloud proxy can cause a crash

Got this stack trace when refreshing and got a 404 from the cloud proxy (not connected). This is due to dereferencing getResp when it is nil on this [line](https://github.com/vmware/terraform-provider-vra/blob/master/vra/data_source_region_enumeration.go#L54).
```
panic: runtime error: invalid memory address or nil pointer dereference
2019-10-26T15:25:37.645-0700 [DEBUG] plugin.terraform-provider-vra: [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1c12c4d]
2019-10-26T15:25:37.645-0700 [DEBUG] plugin.terraform-provider-vra:
2019-10-26T15:25:37.645-0700 [DEBUG] plugin.terraform-provider-vra: goroutine 46 [running]:
2019-10-26T15:25:37.645-0700 [DEBUG] plugin.terraform-provider-vra: github.com/vmware/terraform-provider-vra/vra.dataSourceRegionEnumerationRead(0xc000249d50, 0x1ce9060, 0xc0008b4600, 0xc000249d50, 0x0)
2019-10-26T15:25:37.645-0700 [DEBUG] plugin.terraform-provider-vra: 	/Users/mark/github/vmware/terraform-provider-vra/vra/data_source_region_enumeration.go:55 +0x37d
```
Error returned from POST /iaas/api/cloud-accounts-vsphere/region-enumeration
```
{"message":"The connection with cloud proxy is lost. Please make sure the agent is running in the cloud proxy virtual appliance or restart the agent if problem persists.","statusCode":404,"errorCode":0,"serverErrorId":"3a7cfcf8-edd2-4618-a39a-e35940a11f04","documentKind":"com:vmware:xenon:common:ServiceErrorResponse"}
```

--- a/vra/data_source_region_enumeration.go
+++ b/vra/data_source_region_enumeration.go
@@ -1,8 +1,6 @@
 package vra
 
 import (
-	"log"
-
 	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
 	"github.com/vmware/vra-sdk-go/pkg/client/cloud_account"
 	"github.com/vmware/vra-sdk-go/pkg/models"
@@ -50,8 +48,6 @@ func dataSourceRegionEnumerationRead(d *schema.ResourceData, meta interface{}) e
 		Password: withString(d.Get("password").(string)),
 		Username: withString(d.Get("username").(string)),
 	}))
-
-	log.Printf("%v %v\n", getResp.Payload, err)
 
 	if err != nil {
 		return err

